lib/siwapp_web/live/invoices_live/index.ex

```diff
@@ -0,0 +1,42 @@
+defmodule SiwappWeb.InvoicesLive.Index do
+  use SiwappWeb, :live_view
+  alias Siwapp.Invoices
+
+  def mount(_params, _session, socket) do
+    {:ok,
+     socket
+     |> assign(:invoices, Invoices.list())
+     |> assign(:checked, MapSet.new())}
+  end
+
+  def handle_event("click_checkbox", data, socket) do
```

Comment:
as a convention what you call here data is usually called params. Please rename that.